server-configs/config.json

```diff
@@ -1,29 +1,29 @@
 {
-	"apiVersion": "v3",
+	"apiVersion": "pr174",
```

Comment:
Should this be in the PR? (:

Reply:
No, but the apiVersion should be bumped to v4.